node-wasm/src/worker.rs

```diff
@@ -41,7 +41,7 @@ pub enum WorkerError {
 }
 
 #[wasm_bindgen]
-struct NodeWorker {
+pub(crate) struct NodeWorker {
```

Comment:
actually this should just be `pub`, as anything we have `#[wasm_bindgen]` on, so our rust wasm api matches js one. Tho we can do it separately if you want as it's 'breaking' :stuck_out_tongue: